.github/workflows/cifuzz.yml

```diff
@@ -1,31 +0,0 @@
-name: CIFuzz
-on: [pull_request]
```

Comment:
Could also just disable the `on` line for now.

Reply:
Done, with comment.